pkg/agent/config/config.go

```diff
@@ -27,6 +28,8 @@ type Config struct {
 	LogFilePath *string `envconfig:"DAYTONA_AGENT_LOG_FILE_PATH"`
 	Server      DaytonaServerConfig
 	Mode        Mode
+	Image       string `envconfig:"DAYTONA_PROJECT_IMAGE"`
```

Comment:
Both of these props are unnecessary since the agent has a `getProject` method which returns the project from the API. These props can be extracted from that.

Reply:
User prop is neede in order to load the config - that is necessary to get the api endpoint

Reply:
Why is it necessary to load the config? 

The API endpoint is available to the agent regardless of this.

Reply:
I've refactored the configuration loading in the agent and reverted the env var changes.